core/src/main/java/io/grpc/transport/AbstractStream.java

```diff
@@ -348,7 +348,7 @@ protected final void onSendingBytes(int numBytes) {
    *
    * @param numBytes the number of bytes that were sent.
    */
-  protected final void onSentBytes(int numBytes) {
+  public void onSentBytes(int numBytes) {
```

Comment:
To clarify, I would imagine something like NettyServerStream override onSentBytes to be public, but it would remain protected here.